.github/workflows/test.yml

```diff
@@ -61,8 +61,86 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - uses: pre-commit/action@v3.0.1
 
+  test_mac:
```

Comment:
Can't this be part of the matrix in the tests above, to avoid so much repetition?

Reply:
Done